package.json

```diff
@@ -21,5 +21,8 @@
   "bugs": {
     "url": "https://github.com/luisdanielvcord/cypress-training/issues"
   },
-  "homepage": "https://github.com/luisdanielvcord/cypress-training#readme"
-}
+  "homepage": "https://github.com/luisdanielvcord/cypress-training#readme",
+  "devDependencies": {
+    "cypress": "^9.6.1"
+  }
+}
```

Comment:
EOF. Add one blank line to end of the file to remove this warning:
![image](https://user-images.githubusercontent.com/10209180/169817293-5460321a-6fbb-4c00-b252-792b1dc0c1f6.png)

You can configure visual studio to do so:
https://stackoverflow.com/a/44704969

Reply:
Thank you, a commit with the changes has been pushed.